### ISSUE TITLE
`/subscriptions`: Fix focus outline for view tabs

### ISSUE THREAD

**Issue by mashikag** [Automattic/wp-calypso#76213]
Currently, the tabs have no focus outline at all.

## Related Discussions
pdDOJh-1Kg-p2

**Comment by yansern** [comment]
Ready to review: https://github.com/Automattic/wp-calypso/pull/76882
